ydb/core/kqp/federated_query/kqp_federated_query_helpers.cpp

```diff
@@ -360,6 +364,16 @@ namespace {
                 }
                 return NThreading::MakeFuture<TGetSchemeEntryResult>(res);
             });
+    }
+
+    NThreading::TFuture<TGetSchemeEntryResult> GetSchemeEntryType(
+        const std::optional<TKqpFederatedQuerySetup>& federatedQuerySetup,
+        const TString& endpoint,
+        const TString& database,
+        bool useTls,
+        const TString& structuredTokenJson,
+        const TString& path) {
+        return GetSchemeEntryTypeImpl(NActors::TActivationContext::ActorSystem(), federatedQuerySetup, endpoint, database, useTls, structuredTokenJson, path, false);
```

Comment:
Предлагаю вызвать CanonizePath от database перед дописывание /Root, чтобы гарантировать наличие '/' в начале:


```suggestion
        return GetSchemeEntryTypeImpl(NActors::TActivationContext::ActorSystem(), federatedQuerySetup, endpoint, CanonizePath(database), useTls, structuredTokenJson, path, false);
```